### PR TITLE
Fix/86

### PR DESCRIPTION
**Pull request by EmilioTR**
Fixes issue #86

### CODE REVIEW

File: cypress/e2e/templated-query.cy.js [hunk 1]
```diff
@@ -56,4 +56,83 @@ describe("Templated query", () => {
...
+
+    // Check that the page loaded and that we can see the correct data
+    cy.contains("Finished in:");
+    cy.get('.column-name').find('span').contains("Johann Sebastian Bach");
+
+    // Check if the button to make a new query exists and use it
+    cy.get('button').contains("Change Variables").should("exist");
+    cy.get('button').contains("Change Variables").click();
+
+    // Making sure we get the form te enter new variables
```
Comment (mvanbrab) [inline cypress/e2e/templated-query.cy.js:81]: typo: to

File: cypress/e2e/templated-query.cy.js [hunk 2]
```diff
@@ -56,4 +56,83 @@ describe("Templated query", () => {
...
+    cy.get('form').should("exist");
```
Comment (mvanbrab) [inline cypress/e2e/templated-query.cy.js:82]: Better check to see if it is a form containing "genre". You never know what other forms could come later.

File: cypress/e2e/templated-query.cy.js [hunk 3]
```diff
@@ -56,4 +56,83 @@ describe("Templated query", () => {
...
+    cy.get('button').contains('Query').click();
+
+    // Check that we see the correct message
+    cy.get('span').contains("The result list is empty.").should("exist");
+
+    cy.get('button').contains("Change Variables").should("exist");
+    cy.get('button').contains("Change Variables").click();
+
+    // Making sure we get the form te enter new variables
+    cy.get('form').should("exist");
```
Comment (mvanbrab) [inline cypress/e2e/templated-query.cy.js:134]: Same remarks here.

**Review by mvanbrab — COMMENTED** [review]
Few details in the tests...

File: cypress/e2e/templated-query.cy.js (hunk 1 above)
Comment (EmilioTR) [reply cypress/e2e/templated-query.cy.js]: removed the two typo's

File: cypress/e2e/templated-query.cy.js (hunk 2 above)
Comment (EmilioTR) [reply cypress/e2e/templated-query.cy.js]: updated this

**Pull request merged**